### PR TITLE
add transition to contact form button

### PR DESCRIPTION
**Pull request by Hector096**
Milestone 1: Configure Js linters
Milestone 2: Add mobile menu template 
Milestone 3: Add Button functions for the open and close menu.

PS: Due to merging conflicts I have merged it manually and opened a new pull request for the review purpose.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -4,7 +4,6 @@
   margin: 0;
   padding: 0;
   scroll-behavior: smooth;
-  /* outline: 1px solid red; */
 }
```
Comment (carlos-ssh) [inline styles.css:8]: Kindly add the same style provided in your Figma template (Typography):
Screen Shot 2021-08-31 at 4 54 21

As well improve your project giving the same font 
Screen Shot 2021-08-31 at 4 55 09
color than you style requested:

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Hector096 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Hector096) [reply styles.css:8]: Actually, due to the accessibility factors, I have changed the colour to black. hope it's fine.

Comment (carlos-ssh) [reply styles.css:8]: That accessibility it's mentioned on your project requirements?

**Review by carlos-ssh — APPROVED** [review]
Hi @Hector096 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Hector096) [reply styles.css:8]: No not on this project, but I did that in last week project, so I thought I have to keep that in mind.

Comment (carlos-ssh) [reply styles.css:8]: Don't worry!... Keep coding hard bro! :D

**Pull request merged**